pkg/metrics/utils/utils.go

```diff
@@ -109,6 +109,14 @@ func collectEnergy(ch chan<- prometheus.Metric, instance interface{}, metricName
 	}
 }
 
+func convertUnit(metricName string, val uint64) float64 {
```

Comment:
It might be clearer to have function convertU64ToFloat. 
Then move the logic if microseconds to milliseconds outside after the conversion and define a clear constant like MilliMicroConversionFactor float64 = 1000. 

Or, we can see single function but changing the function name to clarify the task such as convertResUtilToPromValue

Reply:
I agree, but let's fix this in another PR